src/test/java/cat/udl/eps/entsoftarch/webingogeiapi/steps/EditGameStepDefs.java

```diff
@@ -0,0 +1,212 @@
+package cat.udl.eps.entsoftarch.webingogeiapi.steps;
+
+import cat.udl.eps.entsoftarch.webingogeiapi.domain.Game;
+import cat.udl.eps.entsoftarch.webingogeiapi.repository.GameRepository;
+import com.jayway.jsonpath.JsonPath;
+import cucumber.api.PendingException;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.skyscreamer.jsonassert.comparator.JSONComparator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+
+import java.time.ZonedDateTime;
+
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.is;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static sun.audio.AudioPlayer.player;
+
+
+public class EditGameStepDefs {
+
+    @Autowired
+    private final StepDefs stepDefs;
+    GameRepository gameRepository;
+    String idc;
+
+    public EditGameStepDefs(StepDefs stepDefs) {
+        this.stepDefs = stepDefs;
+    }
+
+    @Then("^The game with id (\\d+) has not been edited$")
+    public void theGameWithIdHasNotBeenEdited(int id) throws Exception {
+        stepDefs.result = stepDefs.mockMvc.perform(
```

Comment:
This code retrieves the game but does not check there are no changes compared to something else